src/firefly/js/metaConvert/LsstSdssRequestList.js

```diff
@@ -76,39 +94,41 @@ function makeCoaddReqBuilder(table, rowIdx) {
  * @param threeColorOps
  * @return {{}}
  */
-export function makeLsstSdssPlotRequest(table, row, includeSingle, includeStandard, threeColorOps) {
+export function makeLsstImagePlotRequest(table, row, includeSingle, includeStandard, threeColorOps) {
 
     const retval= {};
     var builder;
+    var plotId;
     if (getCellValue(table, row, 'scienceCcdExposureId')) {
         builder= makeCcdReqBuilder(table,row);
+        plotId = 'lsst-sdss-';
     }
     else {
-        builder= makeCoaddReqBuilder(table,row);
-        //return {single:null, standard:[]};
+        builder= makeCoadReqBuilder(table, row);
+        plotId = 'lsst-coadd-';
+
     }
     const filterId= Number(getCellValue(table, row, 'filterId'));
     const filterName= getCellValue(table, row, 'filterName');
 
     if (includeSingle) {
-        retval.single= builder('lsst-sdss-'+filterName,filterName, filterName);
+       retval.single= builder(plotId+filterName,filterName, filterName);
     }
 
     if (includeStandard) {
         retval.standard= [
-            builder('lsst-sdss-u', 'u', 'u'),
-            builder('lsst-sdss-g', 'g', 'g'),
-            builder('lsst-sdss-r', 'r', 'r'),
-            builder('lsst-sdss-i', 'i', 'i'),
-            builder('lsst-sdss-z', 'z', 'z'),
+            builder(plotId+'u', 'u', 'u'),
+            builder(plotId+'g', 'g', 'g'),
+            builder(plotId+'r', 'r', 'r'),
+            builder(plotId+'i', 'i', 'i'),
+            builder(plotId+'z', 'z', 'z'),
```

Comment:
the plotId should be the same for both types of plots.  No reason to have different id's, it will only hold extra memory.